### DISCUSSION TITLE
nginx.ingress.kubernetes.io/ssl-redirect重定向死循环问题

### DISCUSSION THREAD

**Discussion by hoocoo** [alibaba/higress#886] [Q&A] [answered]
higress前面有alb，现在想在higress配置http重定向到https出现了如下问题，请教下有没有解决方案？
![image](https://github.com/alibaba/higress/assets/31276333/04261e94-8574-4680-b833-9fd2c8d6739e)

**Comment by johnlanni** [comment]
把higress上配置的强致跳转去掉，在alb上配置

**Reply by hoocoo** [reply]
我知道alb可以配。我的意思前面有alb的情况是不能用higress配置了对吗？

**Reply by CH3CHO** [reply, marked as answer]
问题在于，TLS卸载是谁在做？如果是ALB在做，那么Higress拿到的就是HTTP请求。如果又在Higress上配置了强制HTTPS，势必会导致死循环跳转。
